### PR TITLE
Add license, contributing guidelines, update readme

### PR DESCRIPTION
**Pull request by Hwesta**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+In all cases, contributors who sign the Contributor's Agreement retain full rights to use their original contributions for any other purpose outside of Archivematica, while enabling Artefactual Systems, any successor organization which may eventually take over responsibility for Archivematica, and the wider Archivematica community to benefit from their collaboration and contributions in this open source project.
+
+[Artefactual Systems](http://artefactual.com) has made the decision and has a proven track record of making our intellectual property available to the community at large.
+By standardizing contributions on these agreements the Archivematica intellectual property position does not become too complicated.
+This ensures our resources are devoted to making our project the best they can be, rather than fighting legal battles over contributions.
+
+### How do I send in an agreement?
+
+Please print out, read, sign, and scan the [contributor agreement](https://wiki.archivematica.org/images/2/25/Contributor_agreement.txt) and email it to agreement@artefactual.com
```
Comment (sevein) [inline CONTRIBUTING.md]: Can we use https://www.archivematica.org/contributoragreement.txt instead?

**Comment by sromkey** [comment]
This PR looks good to me, with @sevein's comment above.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Hwesta) [inline CONTRIBUTING.md]: TBH I copied this whole section from https://wiki.archivematica.org/Contributor_Agreement but I like the suggested link better.

Comment (Hwesta) [inline CONTRIBUTING.md]: Does it matter that this will be different from the link the other contributing.mds use?

**Comment by Hwesta** [comment]
I think I'll merge this as-is, and there can be a later discussion about where the contributor's agreement information & agreement itself should live.

**Pull request merged**